.github/ISSUE_TEMPLATE/02--issues-with-software-on-platforms.yml

```diff
@@ -1,4 +1,4 @@
-name: Report issues with software on freeCodeCamp.org's platforms
+name: Issue - User Interface (UX) on our Platforms
```

Comment:
```suggestion
name: Issue - User Experience (UX) on our Platforms
```
:eyes:

Reply:
TL;DR: This is intentional

I wanted to use all the common terms UI, UX, and the full form of the most common of both User Interface, all in a concise and tiny title.

The goal is people should not need to read or think.

Reply:
To be honest, I think this creates a bit of confusion. 🤔 
When a term is followed by an acronym in brackets, readers associate the two.

Someone who isn't already familiar with "UX" will think "UX" is short for User Interface. (Which isn't unlikely given the target audience of freeCodeCamp.)

People that know what "UX" means, will likely open a PR or issue to suggest changing it. (Such as this suggestion that was made just now.)

Reply:
Yes - Once in a blue moon, people may try to change it as a nit-pick. You correctly identified that. Maybe dropping the acronym is the best.